src/cljc/athens/common_events.cljc

```diff
@@ -203,6 +203,23 @@
                      :value value}}))
 
 
+(defn build-page-reindex-left-sidebar
```

Comment:
We don't really have this event anymore, right?

Reply:
Removed.